File: Silver/All/SharedUI/SharedUI.Shared/MainWindowController.Cocoa.swift
#if TOFFEE

@IBObject
extension MainWindowController : NSWindowController {

	public init() {
		super.init(windowNibName: "MainWindow")
		setup()
	}

	open override func windowDidLoad() {
		super.windowDidLoad()

		//  Implement this method to handle any initialization after your window controller's
		//  window has been loaded from its nib file.
	}


	//
	// Add Cocoa-specific code here
	//
}

#endif

File: Mercury/All/SharedUI/SharedUI.Shared/MainWindowController.Cocoa.vb
#If TOFFEE Then
<IBObject>
Public Partial Class MainWindowController
  Inherits NSWindowController

  Public Sub New()
	MyBase.New(windowNibName: "MainWindow")
	setup()
  End Sub

  Public Overrides Sub windowDidLoad()
	MyBase.windowDidLoad()
	'  Implement this method to handle any initialization after your window controller's
	'  window has been loaded from its nib file.
  End Sub

  '
  ' Add Cocoa-Specific code here
  '

End Class
#End If